PWGCF/EbyEFluctuations/Tasks/RobustFluctuationObservables.cxx

```diff
@@ -44,18 +44,30 @@ using namespace o2;
 using namespace o2::framework;
 using namespace o2::framework::expressions;
 
-// declaratives to orginise initialization and filling for histograms with the standard set of cuts
-#define FILL_QA_HIST_2D(cutId, hName, x, y...)                      \
-  if (!mHist.count(hName)) {                                        \
-    cout << "AHTUNG! no key " << hName << " in mHist map!" << endl; \
-    return;                                                         \
-  }                                                                 \
-  pHist[cutId][mHist[hName]]->Fill(x, y);
+// declaratives to orginise initialization and filling of histograms for various sets of cuts
+#define FILL_QA_HIST_1D(cutId, hName, x...)                           \
+  if (!mHist1D.count(hName)) {                                        \
+    cout << "AHTUNG! no key " << hName << " in mHist1D map!" << endl; \
+    return;                                                           \
+  }                                                                   \
```

Comment:
Two things to comment here
- please, no `cout` nor `endl`. Use `LOGF` instead
- using `return` in a preprocessor macro it is not safe. As this is a full error condition I would suggest that in line with the previous bullet use `LOGF(fatal, "The fatal message you want");`